site/content/flowcharts/lonecoder.md

```diff
@@ -24,5 +24,62 @@ graph TD;
   class Link,B,C,F linkedBox
 {{< /mermaid >}}
 
-To read about this workflow in detail, please see
-[single developer process](/dev-review/recipes/lonecoder)
+### Find a reviewer
+
+There is no code review without (a) reviewer(s). The good news is,
+almost any researcher who code is a good candidate for being
+one. 
+
+Potential reviewers are not far: in your own research group, others
+you collaborate with or your local Research Software Engineering
+group. They can also be found in communities outside your institution
+or specific research domain. In the [Find a reviewer](https://researchcodereviewcommunity.github.io/dev-review/recipes/find_a_reviewer/) phase, we look at
+different pools of potential reviewers, what makes a suitable
+reviewer, and ways of getting in touch.
+
+### Meet and agree on objectives
+
+A code review can improve code in a lot of different ways, but trying
+to do everything at once is rarely effective. In the [Meet and agree
+on
```

Comment:
Will Markdown complain because you have line breaks in the `[link](here)`?

i.e.
```markdown
[Meet and agree
on
objectives](./alink)
```

v.

```markdown
[Meet and agree on objectives](./alink)
```

I forget, but I see that GitHub is not rendering it as a link (in blue)

Reply:
If you look at the "rich diff" then you can see that the link is rendered correctly in Markdown. This is valid Markdown, albeit not brilliant to look at. I'll make a suggestion in a separate comment to improve.